docs/troubleshooting.md

```diff
@@ -109,20 +110,26 @@ If everything seems healthy, and still no packages show up in the UI, it could b
 
 ## OperatorSource failing to download from datastore
 
-If an OperatorSource fails to reconcile for any reason (i.e operators in that particular OperatorSource fail to show up in the UI), it might be helpful first inspect the `STATUS` and the `MESSAGE` of the OperatorSource with `oc get opsrc -n openshift-marketplace`. You can also inspect the log messages of the marketplace-operator for detailed information about the OperatorSource. 
+If an OperatorSource fails to reconcile for any reason, first inspect the `STATUS` and the `MESSAGE` of the OperatorSource with `oc get opsrc -n openshift-marketplace`. You can also inspect the log messages of the marketplace-operator with `oc logs <name-of-marketplace-operator> -n openshift-marketplace` for detailed information about the OperatorSource. 
 
-Note: A common reason for an OperatorSource to not reconcile correctly has been the use of the wrong [end point](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8). Please make sure that the end point is specified as `https://quay.io/cnr`.
+A common reason for an OperatorSource to not reconcile correctly has been the use of the wrong [end point](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8). Please make sure that the end point is specified as `https://quay.io/cnr`.
+
+If you are creating a new OperatorSource with your own applications by following the steps mentioned [here](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#create-your-quay-app-registry-repository), private applications in the OperatorSource are allowed. To create an OperatorSource with private applications, please follow the steps listed [here](https://github.com/operator-framework/operator-marketplace/blob/master/docs/how-to-authenticate-private-repositories.md).
```

Comment:
I am not sure what you are trying to convey here. How about:
`If there are private app-registry repositories in your namespace, not specifying the authenticationToken in the OperatorSource CR will result in them not being listed. Please follow the steps here for adding the token to the CR.`
Moreover this should come under the new section you added.